drivers/npmx/npmx_shell.c

```diff
@@ -232,6 +232,43 @@ static bool check_instance_index(const struct shell *shell, const char *instance
 	return true;
 }
 
+static npmx_adc_t *adc_instance_get(const struct shell *shell)
+{
+	npmx_instance_t *npmx_instance = npmx_instance_get(shell);
+
+	return npmx_instance ? npmx_adc_get(npmx_instance, 0) : NULL;
+}
+
+static bool check_error_code(const struct shell *shell, npmx_error_t err_code);
+static void print_get_error(const struct shell *shell, const char *str);
```

Comment:
To avoid declaration of static functions before their definition, I would put function getting instance of a driver after definition of `check_error_code`. I only wanted function from the first commit to be put before that function.